frontend/src/queries/types.ts

```diff
@@ -56,6 +56,8 @@ export interface QueryContext<Q extends QuerySchema = QuerySchema> {
     ignoreActionsInSeriesLabels?: boolean
     /** Transform dataTableRows after they are created (e.g., to add date labels) */
     dataTableRowsTransformer?: (rows: DataTableRow[]) => DataTableRow[]
+    /** Compare filter for Web Analytics queries */
+    compareFilter?: any
```

Comment:
TODO: remove it and make web analytics queries use the same pattern as product analytics